vercel-api/src/dashboard/styles/responsive.css

```
@@ -0,0 +1,630 @@
+  
+  .chart-content {
+    height: 200px;
+  }
+  
+  .activity-content {
+    max-height: 250px;
+  }
+}
+
+/* High DPI / Retina Displays */
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)